numpy/typing/_callable.py

```diff
@@ -170,7 +172,7 @@ class _UnsignedIntBitOp(Protocol[_NBit_co]):
         @overload
         def __call__(self, __other: bool) -> unsignedinteger[_NBit_co]: ...
         @overload
-        def __call__(self, __other: int) -> signedinteger[Any]: ...
```

Comment:
`type(np.uint32(1) & 1)` is `int64`, but this indicates it would be `int32`.

Reply:
Good call, fixed again as of https://github.com/numpy/numpy/pull/17514/commits/0dbc4aa45f8b5122094c3a929320ed79a065b4e6 .